src/sage/matroids/unpickling.pyx

```diff
@@ -28,13 +28,15 @@ AUTHORS:
 from sage.data_structures.bitset_base cimport *
 import sage.matroids.matroid
 import sage.matroids.basis_exchange_matroid
-from sage.matroids.minor_matroid import MinorMatroid
-from sage.matroids.dual_matroid import DualMatroid
-from sage.matroids.circuit_closures_matroid cimport CircuitClosuresMatroid
 from sage.matroids.basis_matroid cimport BasisMatroid
-from sage.matroids.linear_matroid cimport LinearMatroid, RegularMatroid, BinaryMatroid, TernaryMatroid, QuaternaryMatroid
+from sage.matroids.circuit_closures_matroid cimport CircuitClosuresMatroid
```

Comment:
Let's not needlessly change the order of things, especially to something just as random.